experiments/05B.md

```diff
@@ -0,0 +1,71 @@
+# Experiment 4: Creating a UI playground and a UI annotation app
+
+
+## Objective
+
+Creating a playground for the users to test our model. Also, creating an annotation app for volunteers to annotate the output of the model and the output of the segmentation algorithm. These apps need to be independent of the model and platform used to run the model: they run locally and on the cloud and the model can be changed easily as it is stored on HuggingFace.
+
+---
+
+## Experiment setup
+
+- Accelerator used for the experiment: GPU Tesla T4
+- Dataset used for the experiment: `Gen_Ref_AbdulBasit.csv`[[1]] as output to annotate and `ayah_text_clear(1).csv`[[2]] as the reference
+- Python modules used: `Gradio`, `Difflib`, and `Torchaudio`
+
+---
+
+
+## APP 1: UI playground
+
+### Objective
+This app is a playground for the users to test our model using their audio files [mostly using the microphone] and for us to see the results and the difference between the original and finetuned model on the fly while developing the model.
+
+### Picture
+ <p align="center">
+     <img src="./media/playgroundUI.png" alt="<alt>">
+ </p>
+ <p align="center">
+    <span style="color: #888888;"> _Figure 05B1_ : UI Playground</span>
+ </p>
+
+### code 
+As the code is too long to be included in the report, it can be found in this [colab notebook][3] so you can run it and test it yourself.
+
+### Bugs
+- The app needs to change the color of the background of the diff element in the app for usability improvements.
+
+---
+
+## APP 2: UI annotation app
+
+### Objective
+This app is an annotation app for a volunteer to annotate the output of the model and the output of the segmentation algorithm: generated_part_nth and segmented_part_nth are mapped using the algorithm but the user can change the mapping if it is wrong.
+
+
+### Picture
+ <p align="center">
+     <img src="./media/AnnotatorUI.png" alt="<alt>">
+ </p>
+ <p align="center">
+    <span style="color: #888888;"> _Figure 05B2_ : UI Annotation App</span>

```

Comment:
My bad I made a mistake in the guidelines, I will make sure it's fixed before the next experiment. 
```suggestion
    <span style="color: #888888;"> <em>Figure 05B2</em> : UI Annotation App</span>
```
Also make sure to update the title of the other figure accordingly.